packages/protocol/test/layer2/LibEIP1559.t.sol

```diff
@@ -98,23 +98,23 @@ contract TestLibEIP1559 is Layer2Test {
         assertEq(baselineBasefee, basefee);
     }
 
-    /// forge-config: layer2.fuzz.runs = 1000
-    /// forge-config: layer2.fuzz.show-logs = true
+    /// forge-config: default.fuzz_runs = 1000
```

Comment:
I think we should also change `layer_2.fuzz_runs` to `default.fuzz_runs`, no?

Reply:
Good point. Rename both `layer2.fuzz.runs` and `layer2.fuzz.show-logs` to `default.fuzz_runs` and `default.fuzz_runs_show_logs`